tests/integration/docker/ruby/ruby2.5/handler.rb

```diff
@@ -1,5 +1,11 @@
 require 'json'
 
 def hello(event:, context:)
-  { body: JSON.generate({'message' => 'Hello Ruby 2.5!'}), statusCode: 200 }
+  {
+    body: JSON.generate({
+      message: 'Hello Ruby 2.5!',
+      version: RUBY_VERSION,
```

Comment:
Isn't it weird that it says Ruby 2.5 where it really is 2.7?

Reply:
It is ruby2.5 test. I added ruby version test in container to existing ruby 2.5 test ([tests/integration/docker/ruby/ruby2.5](https://github.com/dherault/serverless-offline/tree/4b01ed7c87d5e5bbd0a83d71542514f0e8ec787e/tests/integration/docker/ruby/ruby2.5)), and created a new test ([tests/integration/docker/ruby/ruby2.7](https://github.com/dherault/serverless-offline/tree/4b01ed7c87d5e5bbd0a83d71542514f0e8ec787e/tests/integration/docker/ruby/ruby2.7)) for ruby 2.7.